<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_page</class>
 <widget class="QDialog" name="register_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(153, 193, 241);</string>
  </property>
  <widget class="QLineEdit" name="enter_user_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>288</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>288</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_user_password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>288</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>288</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button_real">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>288</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>288</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
